lib/cog/commands/rule.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Cog.Commands.Rule do
+  use Cog.Command.GenCommand.Base,
+    bundle: Cog.embedded_bundle
+
+  alias Cog.Commands.Helpers
+  alias Cog.Commands.Rule.{List, Create, Drop}
+  require Logger
+
+  @moduledoc """
+  Manages rules for commands.
+
+  USAGE
+    rule [subcommand]
+
+  FLAGS
+    -h, --help  Display this usage info
+
+  SUBCOMMANDS
+    list    List all rules (default)
+    create  Add a rule
+    drop    Drop a rule
+  """
+
+  option "help", type: "bool", short: "h"
```

Comment:
The `Helpers.usage` macro takes care of the help option and the show_usage function.